packages/app/src/cli/services/app-logs/write-app-logs.ts

```diff
@@ -13,9 +13,12 @@ export const writeAppLogsToFile = async ({
   apiKey: string
   stdout: Writable
 }) => {
-  const fileName = `app_logs_${appLog.log_timestamp}.json`
+  const identifier = randomUUID().substring(0, 6)
+
+  const formattedTimestamp = formatTimestampToFilename(appLog.log_timestamp)
+  const fileName = `${formattedTimestamp}_${identifier}.json`
```

Comment:
Just curious: do we get an error if there's already a file with this name? I'm just wondering in the extreme off chance that there's a collision.